crates/ruma-serde/src/urlencoded/ser.rs

```diff
@@ -52,6 +52,7 @@ impl<'input, 'output, Target: UrlEncodedTarget> Serializer<'input, 'output, Targ
 
 /// Errors returned during serializing to `application/x-www-form-urlencoded`.
 #[derive(Clone, Debug, PartialEq, Eq)]
+#[non_exhaustive]
 pub enum Error {
```

Comment:
I kinda dislike this being an enum at all, but that's something that can be refactored separately.

Reply:
What were you thinking?